@ndlib/gatsby-theme-marble/src/components/App/Pages/Portfolio/PortfolioBody/PortfolioEdit/EditItems/index.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import ResponsiveGridList from 'components/Internal/ResponsiveGridList'
+import EditItem from './EditItem'
+import NoItems from 'components/App/Pages/Portfolio/PortfolioBody/PortfolioView/PortfolioDisplay/NoItems'
+
+const EditItems = ({ portfolio, className }) => {
+  const { items } = portfolio
+  return (
+    <div className={`${className} grid`}>
+      {
+        // TODO add "Add Item" button
+
+        items.length > 0 ? <ResponsiveGridList measureBeforeMount>
```

Comment:
May be just me, but I like parentheses to enclose components when I render them after a condition like this, so that the opening and closing tags are equally indented.
```
items.length > 0 ? (
  <ResponsiveGridList measureBeforeMount>
    //stuff
  </ResponsiveGridList>
) : (
  <NoItems />
)
```

Reply:
Made suggested edit.